### PR TITLE
Implement bootstrap into JS code

### PR DESCRIPTION
**Pull request by AndriiSameliuk**
Implement styles in JS code.
Design of Login page:
image

Design of Register page:
image

Design of File List page:
image

Work result of Eslint validator:
image

### CODE REVIEW

**Comment by AndriiSameliuk** [comment]
@dkalinin, PTAL.

File: app/View/Component/FileInfo/index.js [hunk 1]
```diff
@@ -33,23 +24,49 @@ export class FileInfo extends StoreAwareComponent {
...
   _render() {
-    this._root.classList.add('opened');
     this._root.innerHTML = `
-    <div>${this._fileName}</div>
-    <div>${this._fileOwner}</div>
-    <div>${this._fileUploadDate}</div>`;
-    const closedButton = new Button(this._root, 'Close');
+            <div class="details-panel-header">
+                <h4><a href="#">${this.file.name}</a></h4>
+                <button type="button" class="btn btn-default btn-xs remove-button">
```
Comment (dkalinin) [inline app/View/Component/FileInfo/index.js]: All buttons should have titles.

File: app/View/Component/FileInfo/index.js [hunk 2]
```diff
@@ -33,23 +24,49 @@ export class FileInfo extends StoreAwareComponent {
...
-    closedButton.onClick((event) => {
+                        <dt>Owner</dt>
+                        <dd>${this.file.owner}</dd>
+
+                        <dt>Uploaded</dt>
+                        <dd>${this.file.uploadDate}</dd>
+                    </dl>
+                </div>
+            </div>`;
+    const closedButton = this._root.querySelector('.remove-button');
```
Comment (dkalinin) [inline app/View/Component/FileInfo/index.js]: closeButton or closingButton

File: app/View/Component/FileItem/index.js [hunk 3]
```diff
@@ -224,14 +255,17 @@ export class FileItem extends DraggableComponent {
...
-      this._div.addEventListener('drop', (event) => this._onDrop(event));
+      this._li.addEventListener('dragenter', (event) => this.onDragEnter(event, this._li));
+      this._li.addEventListener('dragleave', (event) => this.onDragLeave(event, this._li));
+      this._li.addEventListener('drop', (event) => this._onDrop(event));
       this._li.addEventListener('click', (event) => this._onClick(event));
     }
-    this._fileName.addEventListener('dblclick', (event) => this._onDoubleClick(event));
-    this._fileName.addEventListener('click', (event) => this._onClick(event));
+    if (fileName) {
+      fileName.addEventListener('dblclick', (event) => this._onDoubleClick(event));
```
Comment (dkalinin) [inline app/View/Component/FileItem/index.js]: _onDoubleClick is not needed.

File: app/View/Component/FileListPage/index.js [hunk 4]
```diff
@@ -15,31 +15,152 @@ export class FileListPage extends StoreAwareComponent {
...
+  }
+
+  /**
+   * Processes events which buttons on {@link FileListPage} generates.
+   * @param {HTMLElement} div
+   * @private
+   */
+  _processEvents(div) {
+    const input = div.querySelector('#upload-files');
+    input.multiple = true;
```
Comment (dkalinin) [inline app/View/Component/FileListPage/index.js]: Add this in render().

**Comment by AndriiSameliuk** [comment]
@dkalinin, fixed, PTAL.

**Pull request merged**